### DISCUSSION TITLE
@easepick/amp-plugin shows no year in calendar

### DISCUSSION THREAD

**Discussion by protricker** [easepick/easepick#31] [Q&A] [answered]
Hello. I try to use @easepick/amp-plugin. Month displaying is ok, but year dropdown is empty.
![image](https://user-images.githubusercontent.com/62597533/162249665-f8d3022b-f917-4877-8c85-0db136352c8e.png).
I use it via webpack, but copypaste from example in configurator (https://easepick.com/configurator) doesn't work either.
Code from screenshot
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <script src="https://cdn.jsdelivr.net/npm/@easepick/bundle@1.1.3/dist/index.umd.min.js"></script>
</head>
<body>
<input id="birthday"/>
</body>
<script>
    const picker = new easepick.create({
        element: '#birthday',
        css: [
            "https://cdn.jsdelivr.net/npm/@easepick/bundle@1.1.3/dist/index.css"
        ],
        zIndex: 24,
        autoApply: false,
        AmpPlugin: {
            dropdown: {
                months: true,
                years: true
            }
        },
        plugins: [
            "AmpPlugin"
        ]
    })
</script>
</html>
```
Is there any workaround?

**Comment by protricker** [comment, marked as answer]
If i set minYear and maxYear everything works :)
I guess it default value issue.

**Reply by wakirin** [reply]
Yes. I prepared a fix for this bug.
All default values ​​are defined here.

https://github.com/easepick/easepick/blob/4412ce443472d867c6683a184d7e3c5e56cbf016/packages/amp-plugin/src/index.ts#L19-L24

This fix will be included in 1.1.4
